SillyBusinessInc/SillyBusinessGame
# Origin/feat/#10 enemy spawn area

## Task: #10 Enemy spawn area

Create an area that spawns enemies randomly. Give it some nice inspector params like SpawnSpeed = 10.
Maybe even a mecanic like MaxSpawned = 5, that keeps track of how many enemies of that spawner are alive, and only spawns new once if it is under the max. Think about things that can be handy/cool.

Note that if there are no enemies yet, just let it spawn a prefab of a squire or ball or something. We can replace this prefab with the real enemy prefab later.

## reason:
task is done, i made the script and tested it in the scene which is in mustafa map in dev.
the script i made is in /global/scripts, the name of the file is EnemySpawner.

## How to test:
go to my scene in mustafa map and play it. see if that is what the task described.
you could also take the script and place it on a platform with a enemy prefab in the variables.

### links: 
[ticket](https://github.com/SillyBusinessInc/SillyBusinessGame/issues/10)  
[Documentation link](https://hrnl.sharepoint.com/:w:/r/sites/GameDesignDevelopment/_layouts/15/Doc.aspx?sourcedoc=%7BE7AE02C4-3CF2-49E8-9A49-653A1A85DC97%7D&file=Technical%20Documentation.docx&action=default&mobileredirect=true)  

## Checklist before requesting a review
- [ok ] I have changed the trello ticket to the correct ticket.
- [still need to do] I either removed the documentation link if there is none. Or set the documentation to the correct chapter.
- [ok] I made sure that i found all the edge cases and stuff. And tested it before creating this PR.
